src/modules/core/components/IconTooltip/IconTooltip.tsx

```diff
@@ -28,7 +36,7 @@ const displayName = 'IconTooltip';
 
 const IconTooltip = ({
   icon,
-  iconSize = styles.normalIcon,
+  appearance = { size: 'medium', theme: 'dark' },
```

Comment:
Does the `theme` field do anything?

Reply:
Thanks for the reminder, it is supposed to be used for the tooltip styling, I have updated it now.